tendrl/node_agent/message/handler.py

```diff
@@ -0,0 +1,63 @@
+import gevent.event
+import gevent.greenlet
+from gevent.server import StreamServer
+from gevent import socket
+from gevent.socket import error as socket_error
+from gevent.socket import timeout as socket_timeout
+from io import BlockingIOError
+import os
+import sys
+from tendrl.commons.message import Message
+from tendrl.node_agent.message.logger import Logger
+
+
+RECEIVE_DATA_SIZE = 4096
```

Comment:
@brainfunked When reading from a socket it's always required to specify reading byte size, it means never return until read x bytes from the socket.